src/components/Hero.tsx

```diff
@@ -15,6 +15,8 @@ interface HeroProps {
 
 export const Hero: React.FC<HeroProps> = ({ title, description, images }) => {
   const { alt: imageAlt, path: imagePath } = images[0];
+  // currently the `/` is missing from the path value
```

Comment:
my recommendation is to check if has `/` otherwise you add, it makes more generic and also to break in the future if backend start sending this.

Reply:
can we add a standard for this? as in images we store in the FE will always just have the name
Checking `/` at the beginning and possibly adding it if it's missing won't work for external paths then, so this will also break